src/app/pages/AccountPage/index.tsx

```diff
@@ -147,13 +151,13 @@ export function AccountPage(props: Props) {
             <NavItem
               route={`/account/${address}/active-delegations`}
               label={t('account.subnavigation.activeDelegations', 'Active delegations ({{count}})', {
-                count: stake.delegations.length,
+                count: stake.delegations?.length ?? null!,
```

Comment:
yes, it displays `Active delegations ()`

More effort solutions:
- add another translation string `stake.delegations ? t(account.subnavigation.activeDelegations) : t(account.subnavigation.activeDelegationsNull)`
- use a different field: `count` is used for plurality and needs to be a number, and `null` workaround could break. Make that other field a string, so it can be e.g. `-` too, and update translations